server/src/main/java/tw/waterball/survive/io/server/worldobject/WorldObject.java

```diff
@@ -4,10 +4,35 @@
 
 import java.awt.*;
 
+
 public abstract class WorldObject {
-    private Rectangle body;
-    private World world;
-    private float angle;
+    protected Rectangle body = new Rectangle(0, 0, 0, 0);
+    protected World world;
+    protected float angle;
     private int id;
 
+    public WorldObject() {
```

Comment:
`WorldObject` can default its location, but its body's size should be explicitly passed from the constructor.
As the following, FYI:
```java
public WorldObject(Dimension size) {
    this.body =  new Rectangle(size);
}
```